setup_trinity.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from setuptools import setup
+
+
+setup(
+    name='trinity',
+    # *IMPORTANT*: Don't manually change the version here. Use the 'bumpversion' utility.
+    # NOT CURRENTLY APPLICABLE. VERSION BUMPS MANUAL FOR NOW
+    version='0.1.0-alpha.2',
+    description='The Trinity Ethereum Client',
+    author='Piper Merriam',
+    author_email='piper@pipermerriam.com',
+    url='https://github.com/ethereum/py-evm',
+    include_package_data=True,
+    py_modules=[],
+    install_requires=[
```

Comment:
Related to my comment above, if this was a proper `setup.py` in a subdirectory I assume it would be recognized as such during a `pip install` so that all the install_requires` of trinity could really be moved here rather than delegating to `py-evm[trinity,p2p]`

Reply:
I don't forsee using this file for anything other than releases, hence the stubbed out nature of it and still relying on dependencies declared in the main `setup.py` file.